sdk/ai/ai-inference-rest/README.md

```diff
@@ -8,7 +8,7 @@ Key links:
 
 - [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/ai/ai-inference-rest)
 - [Package (NPM)](https://aka.ms/npm-azure-rest-ai-inference)
-- [API reference documentation](https://aka.ms/AAp1kxa)
+- [API reference documentation](https://learn.microsoft.com/javascript/api/@azure-rest/ai-inference/)
```

Comment:
wondering why the broken link is not detected by initial merged pr ci?

Reply:
My guess is that the aka.ms link was valid at some point but expired.